Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -176,14 +177,24 @@ private void Initialize()
                 Utilities.Editor.InputMappingAxisUtility.CheckUnityInputManagerMappings(Definitions.Devices.ControllerMappingLibrary.UnityInputManagerAxes);
 #endif
 
-                //Enable Input (example initializer)
                 AddManager(typeof(IMixedRealityInputSystem), Activator.CreateInstance(ActiveProfile.InputSystemType) as IMixedRealityInputSystem);
             }
 
+            // If the Speech system has been selected for initialization in the Active speech profile, enable it in the project
+            if (MixedRealityManager.Instance.ActiveProfile.IsSpeechCommandsEnabled)
```

Comment:
We can drop the `MixedRealityManager.Instance.`

Also, let's move this into the `InputSystem` check bc speech cannot be used without it.

Reply:
Done, my bad